src/app/components/PriceChart.tsx

```diff
@@ -316,3 +354,116 @@ export function PriceChart() {
     </>
   );
 }
+
+export function PairInfoTab() {
+  const t = useTranslations();
+  const { pairsList } = useAppSelector((state) => state.pairSelector);
+  const selectedPairAddress = useAppSelector(
+    (state) => state.pairSelector.address
+  );
+  const pairInfo = pairsList.find(
+    (pairInfo) => pairInfo.address === selectedPairAddress
+  );
+
+  if (!pairInfo) {
+    return "Selected pair not found in pairsList";
+  }
+
+  return (
+    <>
+      <div className="sm:p-4 xs:ml-2 md:ml-0 xs:mb-6 md:mb-0 md:pb-0 xs:p-0 text-primary-content">
+        <div className="text-lg font-normal mb-3 xs:pt-3 sm:mt-0 sm:pt-0">
+          {pairInfo.name}
+        </div>
+        <div className="border-b border-b-[rgba(255,255,255,0.08)] pb-6">
+          <LabelAndAddress
+            label={t("pair_resource")}
+            address={pairInfo.address}
+            shortenLength={{ min: 8, max: 20 }}
+          />
+          <LabelAndAddress
+            label={t("order_receipt_address")}
+            address={pairInfo.orderReceiptAddress}
+            shortenLength={{ min: 8, max: 20 }}
+          />
+        </div>
+
+        <div className="flex flex-col sm:flex-row">
+          <CoinInfo token={pairInfo.token1} />
+          <div className="border-b-2 border-[rgba(255,255,255,0.05)] sm:border-r-2 sm:border-b-0 my-4 sm:my-0 sm:mx-8 sm:min-h-[160px] sm:max-h-[200px]"></div>
+          <CoinInfo token={pairInfo.token2} />
+        </div>
+      </div>
+    </>
+  );
+}
+
+interface LabelAndAddressProps {
+  label: string;
+  address: string;
+  shortenLength?: { min: number; max: number };
+}
+
+function LabelAndAddress({
+  label,
+  address,
+  shortenLength,
+}: LabelAndAddressProps) {
+  const minLength = shortenLength?.min || 8;
+  const maxLength = shortenLength?.max || 20;
+
+  return (
+    <>
+      <div className="text-sm tracking-[0.5px] opacity-50 font-normal pt-2 !mb-1">
+        {label}
+      </div>
+      <div className="flex flex-row items-start mb-4">
+        <div className="text-base flex items-center">
+          <span className="mr-2">
+            {shortenString(address, minLength, maxLength, "...")}
+          </span>
+          <CopyIcon targetUrl={address} />
+        </div>
+      </div>
+    </>
+  );
+}
+
+interface Token {
+  iconUrl: string;
+  symbol: string;
+  name: string;
+  address: string;
+}
+
+interface TokenInfoProps {
+  token: Token;
+}
+
+function CoinInfo({ token }: TokenInfoProps) {
+  const t = useTranslations();
+
+  return (
+    <div className="flex flex-col items-start xs:mb-4 mb-4 sm:mb-0 w-[50%]">
+      <div className="flex items-center mb-3 pt-8">
+        <img
+          src={token.iconUrl}
+          alt={token.symbol}
+          className="w-8 h-8 rounded-full"
+        />
+        <p className="pl-2 text-base">
+          {token.name.split(" ")[0]} ({token.symbol})
```

Comment:
Use the full token name. On testnet, it may appear as "Dexter Token", but on mainnet, it is correctly "DeXter".

 ```suggestion
          {token.name} ({token.symbol})
```